packages/uxpin-merge-storybook-preset-addon/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@uxpin/merge-storybook-preset-addon",
+  "version": "1.0.0",
+  "description": "Storybook Preset Addon for UXPin Merge",
+  "main": "preset.js",
+  "repository": "https://github.com/UXPin/uxpin-merge-tools/",
+  "author": "dev@uxpin.com",
+  "license": "GPL-3.0",
+  "private": false,
+  "dependencies": {
+    "@uxpin/merge-cli": "link:/Users/naohiro/xenon/uxpin/uxpin-merge-tools-new/packages/uxpin-merge-cli"
```

Comment:
I'm sure you know about this :arrow_up: but just a reminder in case you forget to remove your local file path...

Reply:
yes! I'll change this to `-dev` package based on this branch right before we merge :)